dockers/docker-database/docker-database-init.sh

```diff
@@ -28,6 +28,19 @@ if [[ $DATABASE_TYPE == "dpudb" ]]; then
     redis_port=`expr 6381 + $DPU_ID`
 fi
 
+# TODO: Check the current platform is DPU
```

Comment:
@Pterosaur can we keep the DPU port derivation based upon the midplane IP outside in a separate service that manages the mid-plane IP configuration